src/components/algorithm/Algorithms.js

```diff
@@ -272,10 +283,16 @@ class Algorithms extends Component {
             {description}
           </Typography>,
         ],
-        author: (
+        type,
+        typeField: (
```

Comment:
I suppose you added these to add a class. I don't think such complicated code should be added only for test sake. It will be a bit more complex for the tests, but maybe you can either check a string over a whole row or detect the correct column depending on the position of the header. We can discuss about it.